robottelo/hosts.py

```diff
@@ -1754,6 +1754,19 @@ def cli(self):
         self._cli._configured = True
         return self._cli
 
+    def install_satellite_or_capsule_package(self):
+        """Install Satellite/Capsule package.
+        Note: Make sure required repos are enabled before using this.
+        """
+        if str(settings.server.version.rhel_version)[0] != '9':
```

Comment:
try sat.os_version.major instead, and afaik this module enable step is only required for el8 now and not for other versions, so it's better to be explicit here
```suggestion
        if self.os_version.major == 8
```